openapi.yml

```diff
@@ -22,7 +30,19 @@ paths:
   /api/v1/orders:
     get:
       summary: Get existing orders.
-      # TODO: Later we could add filters through the url query to only get orders matching specific tokens etc.
+      parameters:
+        - name: owner
+          in: query
+          schema:
+            $ref: "#/components/schemas/Address"
+        - name: sellToken
```

Comment:
For anything that is in the URL, I wouldn't use `camelCase`, I would use kebab case

Reply:
The official [documentation](https://swagger.io/docs/specification/describing-parameters/) uses also camelCase: usderId instead of user_id

Reply:
This is totally subjective. I don't *mind* either but I *prefer* kebab_case.

Reply:
Regarding `kebab-case` for URL query paramaters, I feel that is awkward as they are treated as separate words even though they are a single identifier (although I agree that URL **paths** should use `kebab-case`).

Reply:
Yep, you convinced me. the resource path `kebap` and the params is ok in `camel`